Library/Homebrew/extend/pathname.rb

```diff
@@ -331,7 +331,6 @@ def verify_checksum(expected)
     raise ChecksumMismatchError.new(self, expected, actual) unless expected == actual
   end
 
-  # FIXME: eliminate the places where we rely on this method
```

Comment:
I wonder where we actually rely on this. This should really not be that hard to fix.